geceval/modules/language_switch_module.py

```diff
@@ -0,0 +1,57 @@
+import re
+
+import fasttext
+from huggingface_hub import hf_hub_download
+
+from geceval.modules.gec_module import GECModule
+
+
+class LanguageSwitchModule(GECModule):
```

Comment:
I had an impression that this is almost a duplication LanguageSwitchModule and LanguageStabilityModule - I think at the end the LanguageStabilityModule was used, wasn't it?

Reply:
Right, removed language stability (we used switch instead)